### PR TITLE
[TRA-444] Emit a metric to record market mapper revenue per market

### PR DESCRIPTION
**Pull request by shrenujb**
### Changelist
Emit a metric to record market mapper revenue per market

### Test Plan
Look at datadog

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

## Summary by CodeRabbit

- **New Features**
    - Added metric tracking for fees transferred to the market mapper. 

- **Chores**
    - Removed outdated TODO comment related to monitoring.

### CODE REVIEW

File: protocol/x/subaccounts/keeper/transfer.go [hunk 1]
```diff
@@ -262,8 +264,17 @@ func (k Keeper) DistributeFees(
...
 
+	// Emit a metric for the amount of fees transferred to the market mapper
+	labels := []metrics.Label{
+		metrics.GetLabelForIntValue(metrics.MarketId, int(perpetual.Params.MarketId)),
+	}
+	metrics.AddSampleWithLabels(
+		metrics.MarketMapperRevenue,
+		metrics.GetMetricValueFromBigInt(marketMapperShare),
+		labels...,
+	)
```
Comment (vincentwschau) [inline protocol/x/subaccounts/keeper/transfer.go:275]: What do you plan on doing with this metric? It's emitted per block, but gets sampled, does that mean we get some approx. sum over the blocks or approx. average?

Comment (shrenujb) [reply protocol/x/subaccounts/keeper/transfer.go:275]: Hmm i hope these metrics don't get sampled. At least the IncrCounter and AddSample ones.
How do you know these are sampled?

Comment (shrenujb) [reply protocol/x/subaccounts/keeper/transfer.go:275]: I see we use the same metric to count liquidations and deleveraging: https://github.com/dydxprotocol/v4-chain/blob/dfd4f6798265101bbd52a17c92da3d3a1a87652c/protocol/x/clob/keeper/deleveraging.go#L111

Comment (shrenujb) [reply protocol/x/subaccounts/keeper/transfer.go:275]: Asked around and these metrics are not sampled by default

Comment (vincentwschau) [reply protocol/x/subaccounts/keeper/transfer.go:275]: `AddSample` does do summarization of metrics, but the summary does track the `sum` which is what we need. Can you update the metric name to indicate this, e.g. similar to the deleverage metric with the `distribution` suffix?

**Review by vincentwschau — APPROVED** [review]

**Pull request merged**